external-crates/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2104,6 +2124,61 @@ fn parse_field_annot(context: &mut Context) -> Result<(Field, Type), Box<Diagnos
     Ok((f, st))
 }
 
+// Parse a postfix ability declaration:
+//     "has" <Ability> (, <Ability>)+;
+//  Error if:
+//      * Also has prefix ability declaration
+fn parse_postfix_ability_declarations(
+    infix_ability_declaration_loc: Option<Loc>,
+    abilities: &mut Vec<Ability>,
+    context: &mut Context,
+) -> Result<(), Box<Diagnostic>> {
+    let postfix_ability_declaration =
+        context.tokens.peek() == Tok::Identifier && context.tokens.content() == "has";
+    let has_location = current_token_loc(context.tokens);
+
+    if postfix_ability_declaration
+        && context
+            .env
+            .check_feature(&FeatureGate::PostFixAbilities, None, has_location)
```

Comment:
The package name here is super important as it checks whether or not the package has that edition :)

Reply:
I think you will have to add the package name to the context to do this filtering here.

Reply:
ah I see -- will fix!

Reply:
Plumbed the package name through, and this is now based off of that here.